cognee/infrastructure/llm/structured_output_framework/baml/baml_client/sync_client.py

```
@@ -89,106 +92,30 @@ def parse(self):
-            args={
-                "content": content,
-            },
-        )
-        return typing.cast(
-            types.SummarizedContent, result.cast_to(types, types, stream_types, False, __runtime__)
-        )
+    def AcreateStructuredOutput(
+        self,
+        text_input: str,
+        system_prompt: str,
+        baml_options: BamlCallOptions = {},
```

Ruff (0.13.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)